modules/drip.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+#ifndef DSY_DRIP_H
+#define DSY_DRIP_H
+
+#include <stdint.h>
+#ifdef __cplusplus
+
+namespace daisysp
+{
+/* Drip
```

Comment:
Not a doxygen comment tag -- update to /**

Also worth mentioning that the 'name' section of the documentation gets autopopulated by what's being parsed. So the first line of the doxygen comments should be the brief description. (same applies to the other function documentation here, and in allpass).

Right now it'll looks like

# Drip
# Drip

Something about drip